packages/ui/src/Banner.tsx

```diff
@@ -16,141 +94,107 @@ export const hideBanner = (id: string): Promise<void> => {
-    //       text={customButtonProps?.text ?? ""}
-    //       onClick={onClick}
-    //     />
-    //   );
-    // } else
-    if (type === ACTION) {
-      return (
-        <Box alignItems="center" display="block" justifyContent="center" width={40}>
-          {/* accessibility-hint handled in IconButton component */}
-          {/* eslint-disable-next-line react-native-a11y/has-accessibility-hint */}
-          <IconButton
-            accessibilityHint="Follow banner action"
-            accessibilityLabel="arrow-right"
-            iconName="arrow-right"
-            onClick={(): void => onClick?.()}
-          />
-        </Box>
-      );
-    } else if (type === DISMISS) {
-      return (
-        <Box alignItems="center" display="block" justifyContent="center" width={40}>
-          <IconButton
-            accessibilityHint="Banner is dismissed"
-            accessibilityLabel=""
-            iconName="circle-xmark"
-            onClick={dismiss}
-          />
-        </Box>
-      );
-    } else {
-      return null;
-    }
-  };
-
   if (!show) {
     return null;
   }
 
-  if ((type === ACTION && !onClick) || (type === CUSTOM_BUTTON && !onClick)) {
-    console.warn("Banners with type action require an onClick property.");
-  }
-
-  const negativeMargin = negativeXMargin * -4;
-
   return (
-    <Box
-      color={color}
-      dangerouslySetInlineStyle={{
-        __style: {
-          marginLeft: negativeMargin,
-          marginRight: negativeMargin,
-        },
+    <View
+      style={{
+        alignItems: "center",
+        backgroundColor: theme.surface[bgColor],
+        height: "auto",
+        width: "100%",
+        margin: "auto",
+        flexDirection: "row",
+        borderRadius: theme.radius.default as any,
+        minHeight: theme.spacing.xl as any,
+        padding: theme.spacing.xs as any,
       }}
-      direction="row"
-      justifyContent="between"
-      paddingX={3}
-      paddingY={2}
-      rounding={shape}
-      shadow
-      width={width || Unifier.utils.dimensions().width || "100%"}
-      onClick={type === DISMISS ? dismiss : undefined}
     >
-      {iconName && (
-        <Box justifyContent="center" width={32}>
-          <Icon color={textColor} iconName={iconName} size="lg" />
-        </Box>
-      )}
-      <Box alignItems="center" direction="column" flex="shrink" justifyContent="center">
-        <Box paddingY={1}>
-          <Text align="center" bold color={textColor}>
-            {text}
-          </Text>
-        </Box>
-        {subtext && (
-          <Box paddingY={1}>
-            <Text align="center" color={textColor}>
-              {subtext}
-            </Text>
-          </Box>
+      <View
+        style={{
+          flexDirection: "row",
+          flex: 1,
+          alignItems: "center",
+          justifyContent: "center",
+        }}
+      >
+        {Boolean(hasIcon) && (
+          <View style={{paddingRight: 12, paddingLeft: theme.spacing.xs as any}}>
```

Comment:
It's 12px on figma to the right between the icon and the text, and then the xs padding between the edge of the component and the icon over on the left when the text is really long. I will go back and look and see if there is a reason why I did it this way before I merge it in - I also added the xs padding in the parent which makes more sense and was the intention, so this might be able to be pulled out.